File: src/main/java/org/dedda/bratwurst/parse/FileCreateParser.java
package org.dedda.bratwurst.parse;

import org.dedda.bratwurst.lang.io.file.FileCreate;

/**
 * Created by dedda on 1/25/16.
 *
 * @author dedda
 */
public class FileCreateParser extends InstructionParser {

    @Override
    public FileCreate parse(String line, int lineNumber) {
        String variableName = line.substring(1, line.length() - 1);
        return new FileCreate(lineNumber, variableName);
    }

}

File: src/main/java/org/dedda/bratwurst/parse/FileCreateParser.kt
package org.dedda.bratwurst.parse

import org.dedda.bratwurst.lang.io.file.FileCreate

/**
 * Created by dedda on 1/25/16.
 *
 * @author dedda
 */
class FileCreateParser : InstructionParser() {
    override fun parse(line: String, lineNumber: Int): FileCreate {
        val variableName = line.substring(1, line.length - 1)
        return FileCreate(lineNumber, variableName)
    }
}